rest/permissions.py

```diff
@@ -130,3 +134,69 @@ def has_permission(self, request, view):
             return True
         return RepoPermission.manage_repo_users[0] in get_perms(
             request.user, repo)
+
+
+class AddEditMetadataPermission(BasePermission):
+    """Checks add_edit_metadata permission"""
+
+    def has_permission(self, request, view):
+        if request.method in SAFE_METHODS:
```

Comment:
Same comment about this as [here](https://github.com/mitodl/lore/pull/183#discussion_r32950841)